jobs/uaa/templates/tomcat.server.xml.erb

```diff
@@ -59,7 +59,7 @@
       <Host name="localhost" appBase="webapps" unpackWARs="true" autoDeploy="false">
         <Valve className="org.apache.catalina.valves.RemoteIpValve"
                remoteIpHeader="x-forwarded-for"
-               protocolHeader="x-forwarded-proto"
+               protocolHeader="<%= properties.uaa.ssl.protocol_header %>"
```

Comment:
this should use the 

    p('uaa.ssl.protocol_header')

instead of 

    properties.uaa.ssl.protocol_header